core/tests/protractor/learnerDashboard.js

```diff
@@ -217,10 +221,10 @@ describe('Learner dashboard functionality', function() {
 
     // Both creators should be present in the subscriptions section of the
     // dashboard.
-    browser.get(general.LEARNER_DASHBOARD_URL);
+    learnerDashboardPage.get();
     browser.waitForAngular();
     general.waitForSystem();
-    element(by.css('.protractor-test-subscriptions-section')).click();
+    learnerDashboardPage.navigateToSubscriptionsSection();
     browser.waitForAngular();
     expect(element.all(by.css(
```

Comment:
I guess this stuff should also be factored into the learner dashboard page?

Reply:
As it was just after a learnerDashboard.get(), I assumed it to be a part of learner dashboard itself.

Reply:
That's the point I was making, though. If it's part of the learner dashboard page, shouldn't it be refactored into the LearnerDashboardPage object?

Reply:
Yes, thats what I did. Is there something more?

Reply:
What I'm saying is that this line (`expect(element.all(by.css(...))).toMatch(...);`) should be wrapped in a method on the LearnerDashboardPage object, so that this line changes to something like `learnerDashboardPage.someMethodName('creator...')`. It doesn't look like that's been done.

Reply:
Oh, ok, will add that

Reply:
Done